EastAgile/twitterbot
# [f] 883 - option to run public stream or user stream

Description
--
* Allow users to choose public stream or user stream to listen to
* Allow users to pass credentials as command options

How to test
--
* Start responsebot without `--user-stream` flag, it should log saying 'Listening to public stream'
* Start responsebot with `--user-stream` flag, it should log saying 'Listening to user stream'
